gruntfile.js

```diff
@@ -11,7 +11,7 @@ module.exports = function (grunt) {
     authCode = grunt.file.readJSON('secret/auth_code.json')
   } catch (e) {
     if (e.origError.code === 'ENOENT') {
-      console.warn('secret/auth_code.json is not found. CodeSigning is not available.')
+      console.warn('secret/auth_.json is not found. CodeSigning is not available.')
```

Comment:
Why did you change this? Can you share your thought on this?

Reply:
Honestly, it was non-intentional. I'll fix it right away.

Reply:
I would appreciate if you could review this changes.